src/ga/comparison.py

```diff
@@ -0,0 +1,129 @@
+import os
+import sys
+import timeit
+import ga as Ga
+import numpy as np
+import matplotlib.pyplot as plt
+
+from datetime import datetime
+from scipy.spatial.distance import cdist
+
+dir = os.path.join(os.getcwd(), 'src', 'model')
+sys.path.append(dir)
+
+import model as Model  # noqa
+
+TOTAL_DEVICE: int = 8
+
+# DISTANCE: np.ndarray = np.random.random((TOTAL_DEVICE, 2))
+DISTANCE: np.ndarray = np.array([[1, 10], [1, 15], [1, 20], [1, 25], [
+                                1, 30], [1, 35], [1, 40], [1, 45]])
+
+MODEL: Model = Model.Model(total_device=TOTAL_DEVICE,
+                           distances=cdist(DISTANCE, DISTANCE))
+
+network: bool = True
+
+total_penalty: list = []
+
+device_specification: list = []
+
+connection_specification: list = []
+
+each_penalized_transmission_power: np.ndarray = []
+
+if __name__ == '__main__':
+    start_time = timeit.default_timer()
+
+    while network:
```

Comment:
This is where we could use `range` for specific number of iteration for the Monte Carlo Simulation